### PR TITLE
Drop redundant models/views/serializers

### PR DESCRIPTION
**Pull request by rhyw**

### CODE REVIEW

File: openlcs/packages/views.py [hunk 1]
```diff
@@ -987,3 +904,96 @@ class ComponentViewSet(ModelViewSet, PackageImportTransactionMixin):
...
+        ####__Response__####
+
+            HTTP 200 OK
+            Content-Type: application/json
+            [
+                {
+                    "id": 5,
+                    "source": {
+                        "id": 5,
+                        "packages": [],
```
Comment (hanchuntao) [inline openlcs/packages/views.py:927]: Still exist packages ?

Comment (rhyw) [reply openlcs/packages/views.py:927]: ahh these docstring needs to be updated. will fix.

**Comment by rhyw** [comment]
Plan to merge before EOF today. Any more comments leave at your early conveniences, thanks!

**Comment by hanchuntao** [comment]
LGTM

**Comment by huiwangredhat** [comment]
LGTM.

**Pull request merged**